test/es6/default-splitscope.js

```diff
@@ -6,260 +6,622 @@
-            function test(x = eval("var a = 1; let b = 2; const c = 3;")) {
-                if (x === undefined) {
-                // only a should be visible
-                assert.areEqual(1, a, "Var declarations leak out of eval into parameter scope");
-                } else {
-                // none should be visible
-                assert.throws(function () { a }, ReferenceError, "Ignoring the default value does not result in an eval declaration leaking", "'a' is undefined");
-                }
-
-                assert.throws(function () { b }, ReferenceError, "Let declarations do not leak out of eval to parameter scope",   "'b' is undefined");
-                assert.throws(function () { c }, ReferenceError, "Const declarations do not leak out of eval to parameter scope", "'c' is undefined");
-            }
-            test();
-            test(123);
-
-            // Redefining locals
-            function foo(a = eval("var x = 1;")) {
-                assert.areEqual(1, x, "Shadowed parameter scope var declaration retains its original value before the body declaration");
-                var x = 10;
-                assert.areEqual(10, x, "Shadowed parameter scope var declaration uses its new value in the body declaration");
-            }
-
-            assert.doesNotThrow(function() { foo(); }, "Redefining a local var with an eval var does not throw");
-            assert.throws(function() { return function(a = eval("var x = 1;")) { let x = 2; }(); },   ReferenceError, "Redefining a local let declaration with a parameter scope eval var declaration leak throws",   "Let/Const redeclaration");
-            assert.throws(function() { return function(a = eval("var x = 1;")) { const x = 2; }(); }, ReferenceError, "Redefining a local const declaration with a parameter scope eval var declaration leak throws", "Let/Const redeclaration");
-
-            // Function bodies defined in eval
-            function funcArrow(a = eval("() => 1"), b = a()) { return [a(), b]; }
-            assert.areEqual([1,1], funcArrow(), "Defining an arrow function body inside an eval works at default parameter scope");
-
-            function funcDecl(a = eval("function foo() { return 1; }"), b = foo()) { return [a, b, foo()]; }
-            assert.areEqual([undefined, 1, 1], funcDecl(), "Declaring a function inside an eval works at default parameter scope");
-
-            function genFuncDecl(a = eval("function * foo() { return 1; }"), b = foo().next().value) { return [a, b, foo().next().value]; }
-            assert.areEqual([undefined, 1, 1], genFuncDecl(), "Declaring a generator function inside an eval works at default parameter scope");
-
-            function funcExpr(a = eval("var f = function () { return 1; }"), b = f()) { return [a, b, f()]; }
-            assert.areEqual([undefined, 1, 1], funcExpr(), "Declaring a function inside an eval works at default parameter scope");
+    { 
+       name: "Split parameter scope in function definition", 
```

Comment:
Indentation got weird here.  name starts seven spaces indented now instead of eight and the rest of the file seems to follow.